.wake/extension/local-local-chain-1f3157cf-bb77-4e76-965d-1db9c92cfc00.json

```diff
@@ -0,0 +1 @@
d-1db9c92cfc00","displayName":"Local Chain 1","state":{"accounts":[{"address":"0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266","balance":1e+22},{"address":"0x70997970c51812dc3a010c7d01b50e0d17dc79c8","balance":1e+22},{"address":"0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc","balance":1e+22},{"address":"0x90f79bf6eb2c4f870365e785982e1f101e93b906","balance":1e+22},{"address":"0x15d34aaf54267db7d7c367839aaf71a00a2c6a65","balance":1e+22},{"address":"0x9965507d1a55bcc2695c58ba16fb37d819b0a4dc","balance":1e+22},{"address":"0x976ea74026e726554db657fa54763abd0c3a0aa9","balance":1e+22},{"address":"0x14dc79964da2c08b23698b3d3cc7ca32193d9955","balance":1e+22},{"address":"0x23618e81e3f5cdf7f54c3d65f7fbc0abf5b21e8f","balance":1e+22},{"address":"0xa0ee7a142d267c1f36714e4a8f75612f20a79720","balance":1e+22}],"deployment":[],"history":[]},"network":{"wakeDump":{"metadata":{"labels":{},"deployedLibraries":{}},"chainDump":"0x1f8b08000000000000ffed56cd6e23370c7e9739e7a03fea27c7a0e8f6d04351604fc5c2a0282a1e743c637826dd1481dfbdd4d84991c05874d1b914a80c181a4ae24752d447be747998e8f7eefea51b9f0e994fdd7da79e5577d7d1d48f1967be08fee190734b7fe079c1c3713da845f288f36ee80ffd729150221bdbcea6bdf205c0e68484ab82d2d7dad3d3b0fcf966cbf1c47f9c702c387d9735dfb052bcce3b7e269ee75db34f94ef8ea79eb845e22a5ff7c86277afae07e4e3ba499fcf771d124d4fe332b733e2982b1452f2aea02115b3b13ec56c8b250a84d6e8644b4a006baca7b169697a71c0752e1a8c669d5851ca988d7b3595a6725996f9bc4c277c6c46367cc184621d6205677c28399440d6876893c88296788a291efda698e2978e1c35db0a546a1070b2c583cc3229cc15b2288d754b4c4bce5129257b4c4a55341922946292a8b1459b8ace249b694b4cc790b273d1856c43821021c6641461883557e3496597c07f03f31d42a8ff6e88a35e692f7647f9372a1a1ba2b2108dcc5c85a83468f0ca36537534b5408eb20b0ca8f693c34e4e53b5377c0d2aa5908222d0519b42169556148ad21914aba243cb6d8a5bc637a91a52ae9eb321576310ff8025ca291ad6552bcd72a349f94d3183670c4e19cfc17800271905a122b8e02de6a29ae7886953cc24402a148d0099c8f8040431a3f65512ab449db242e18e2d31513107d4ce14e105d25548413b76186b002fcfc52894eb366a4bcc6a532d9e41239618abaf8ed8638ed10423b923299c93317eb3fb14d02c9566b716b0dd87eab50a85987f7be9f68ca5adbc74473cf1b8fc84f37eab3232efd17e1e69e0f952db0a912b1c0b53801250bc870c3e644f549cc662b571a0737251b250dbaada1dd5e25471c9ba201a0ffdf8eac67798b1e0c2bf4ed3b2955f8b54db1969e9a7717ed30b5e58bfeaa0258b01d0d71aad03f649ca5d8d72f3d94516c69084272ca484ad8ca919d87a9b9d69adc08989fbe3326f69eb303dce0fc3341d3651f8fff86f8fdb0de40776903eeee71b0da9883fcf5cdeb6bd6f657d60956368e5939fe581fc800bbef2d2a17fde9256aef47843d9b56dfe91f9173e7dc2f963f3fcb55ff6e5845f71d8f4ddb6eef7d387e85cbae487cbcadfadfacab10f8c348d0f8d84b77beae7f7bc24e4fee5ae9b0e073eadf3f3975bebfbbed58c9e70d8ad2c29e2f16918ce7f017c452e51f90c0000"},"type":"Local Chain","config":{"sessionId":"local-chain-1f3157cf-bb77-4e76-965d-1db9c92cfc00","type":"anvil","uri":"ws://127.0.0.1:54459"}},"stateFingerprint":"3a82e3a6d1ac2e4daae286769e6107faa655d321408b0b1f9df23ede1a63b531","persistence":{"isDirty":false,"isAutosaveEnabled":true,"lastSaveTimestamp":1742780532652}}
```

Comment:
What's this? Probably some tool you're using? Maybe foundry since I see 'anvil'? Probably good to add this to .gitignore and .npmignore